dali/operators/decoder/audio/audio_decoder_op.cc

```diff
@@ -22,21 +23,85 @@ DALI_SCHEMA(AudioDecoder)
 This operator is a generic way of handling encoded data in DALI.
 It supports most of well-known audio formats (wav, flac, ogg).
 
-This operator produces two outputs:
-output[0]: batch of decoded data
+This operator produces two outputs: <br/>
```

Comment:
Do we want to inset <br/>? How it looks inside python when you look into help?

Reply:
I think double new line does the job.